doc/message-registry.yaml

```diff
@@ -0,0 +1,136 @@
+# NOTE:
+# • A code should always have the same meaning forever.
+# • Codes may be retired or introduced in new releases.
+# See the following example of guidelines:
+# https://github.com/haskellfoundation/error-message-index/blob/main/tool-developers.md#code-assignment-recommendations
+
+# TODO: add precise guidelines
+# TODO: fix missing detailed description, examples, resolution
+
+- id: "unrecognized-keysym"
+  code: 107
+  added: UNKNOWN
```

Comment:
1.0.0 will do for any of the really old ones. or "always" :)

Reply:
It is displayed as "1.0.0" in the generated doc, but I prefer to keep “UNKNOWN” in the register. I think it is better to be explicit. Maybe “TODO” is a good alternative.

Reply:
Done: I used ALWAYS and added a comment to give context.